woocommerce/woocommerce-android
# Jetpack CP: Add retry functionality in case of plugin installation failure

### Description
This is the fourth part for the "Jetpack CP: Install Jetpack from app" feature: #4865. 

This PR allows to automatically retry a couple of times in the case of Jetpack installation or activation failure.

It also adds a check whether Jetpack already exists (but inactivated) before attempting to install it.

### Testing instructions

Have a look at the retrying tests in JetpackInstallViewModelTest, and make sure all tests there run correctly.
